mogp_emulator/tests/test_fitting.py

```diff
@@ -61,17 +61,16 @@ def test_fit_GP_MAP_GPU(monkeypatch):
     gp = fit_GP_MAP(gp)
 
     assert isinstance(gp, GaussianProcessGPU)
-    assert_allclose(gp.theta, theta_exp)
+    assert_allclose(gp.theta.data, theta_exp)
```

Comment:
I think this won't work yet with the GPU version, until theta is migrated to be a GPParams, and even then I think it will need `theta.get_data()` rather than `theta.data` won't it?   (But that's fine - lots more needs to be migrated for the GPU version to work!)

Reply:
Yes, I didn't intend to touch any of the GPU tests, but may have accidentally modified this in a global find/replace without realizing. I'll undo this and check if I accidentally affected any other GPU tests.

Reply:
I undid this, and will leave the GPU tests in your hands as you see fit to modify.